src/rofl/common/cthread.cpp

```diff
@@ -298,107 +298,78 @@ cthread::clear_timers()
 {
 	AcquireReadWriteLock lock(tlock);
 	ordered_timers.clear();
-	for (auto it : timers)
-		delete it.second;
-	timers.clear();
 };
```

Comment:
it is actually around quite a lot, so I would be happy to remove them all at once. As well adding a clang-format file might be worth it :)